token/services/identity/common/lm.go

```diff
@@ -251,7 +251,8 @@ func (l *LocalMembership) registerIdentityConfiguration(identity *driver.Identit
 		l.logger.Warnf("failed to load local identity at [%s]:[%s]", identity.URL, err)
 		// Does path correspond to a folder containing multiple identities?
 		if err := l.registerLocalIdentities(identity); err != nil {
-			return errors.WithMessage(err, "failed to register local identity")
+			// we don't return the error so that the token manager can still load
```

Comment:
Should we allow the caller to decide if the error is fatal or not? For example, this function is called by `RegisterIdentity` which can be called at runtime to register an identity. I would expect to get the error if something wrong happens. No?
I agree that when this function is called at `load` time, then we can just print the errors but continue with the processing. What do you think?

Reply:
Good point, changed it now so that when loading all the identities at startup (from database or disk), it won't fail the whole setup if any individual identity fails to load.